Vagrantfile

```diff
@@ -17,6 +17,10 @@ vconfig = {
   'vagrant_ip' => '0.0.0.0',
   'vagrant_memory' => 1024,
   'vagrant_cpus' => 2,
+}
+
+# Default project config.
+pconfig = {
   'beet_home' => '/beetbox',
   'beet_base' => '/var/beetbox',
```

Comment:
@christopher-hopper - doesn't really make much sense to change `beet_home` as this would reinstall `beetbox` into a different path and then reprovision - it's mainly configurable for CI purposes where we don't have a pre-provisioned base image.

Also whilst `beet_base` is configurable we'd discourage it for support reasons as it could break if set to a pre-exisiting path. Basically, this is config that should be abstracted from the user as it shouldn't matter where the project gets mounted to inside the VM as long as it's consistent, no one should be relying on hard coded paths...

I realise it might be rather confusing but in almost every case we have chosen to make things configurable even when we probably should have some project standards, these are some good examples. 

BTW - there's some history behind these decisions -- https://github.com/beetboxvm/beetbox/issues?q=is%3Aissue+is%3Aclosed

I like the PR though, just not exposing these particular vars to end users.